qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/azure.tf

```diff
@@ -4,15 +4,15 @@ provider "azurerm" {
 
 
 module "registry" {
-  source              = "{{ cookiecutter.terraform_modules.repository }}//modules/azure/registry?ref={{ cookiecutter.terraform_modules.rev }}"
+  source              = "./azure/registry"
```

Comment:
I guess `./azure/registry` -> `./modules/azure/registry`?

Reply:
Well spotted. I will run the integration tests after merge anyway to pick up any issues. I tried Azure, but maybe not with the final version of this branch :)